doc/ale.txt

```diff
@@ -2951,6 +2952,7 @@ documented in additional help files.
     jq....................................|ale-json-jq|
     prettier..............................|ale-json-prettier|
     spectral..............................|ale-json-spectral|
+    vscodejson............................|ale-json-vscode|
```

Comment:
Please add an entry for this new tool in the `ale-supported-languages-and-tools.txt` and `supported-tools.md` files.

Reply:
Done.